### PR TITLE
Replaced Coronasafe Network branding to OHC network branding on login page of CARE

### PR DESCRIPTION
**Pull request by siddnikh**
This PR replaces the Coronasafe Network branding to OHC Network branding on the login page of CARE platform.
Also added support for further adding/replacing branding in other parts of the platform. Can be utilised later.

### CODE REVIEW

File: public/config.json [hunk 1]
```diff
@@ -6,6 +6,8 @@
     "analytics_server_url": "https://plausible.10bedicu.in",
     "static_header_logo": "https://cdn.coronasafe.network/header_logo.png",
     "static_light_logo": "https://cdn.coronasafe.network/light-logo.svg",
+    "static_ohc_light_logo": "https://cdn.coronasafe.network/ohc_logo_light.png",
+    "static_ohc_green_logo": "https://cdn.coronasafe.network/ohc_logo_green.png",
     "static_black_logo": "https://cdn.coronasafe.network/black-logo.svg",
     "static_dpg_white_logo": "https://digitalpublicgoods.net/wp-content/themes/dpga/images/logo-w.svg",
     "static_coronasafe_logo": "https://3451063158-files.gitbook.io/~/files/v0/b/gitbook-legacy-files/o/assets%2F-M233b0_JITp4nk0uAFp%2F-M2Dx6gKxOSU45cjfgNX%2F-M2DxFOkMmkPNn0I6U9P%2FCoronasafe-logo.png?alt=media&token=178cc96d-76d9-4e27-9efb-88f3186368e8",
```
Comment (rithviknishad) [inline public/config.json:13]: Shouldn't this be removed?
```suggestion
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Remove usages and references of `static_coronasafe_logo`. Also make a PR for the same for the deploy configurations too: https://github.com/coronasafe/care_deploy_configs

cc: @mathew-alex

File: public/config.json (hunk 1 above)
Comment (siddnikh) [reply public/config.json:13]: I was thinking we could keep it there, in case it needs to be used later?

Comment (rithviknishad) [reply public/config.json:13]: Alright. However, make a PR for the changes made here to the deploy configs repo too.

**Review by siddnikh — COMMENTED** [review]
Removed all references of `static_coronasafe_logo` and replaced with `static_ohc_light_logo`

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**